simulators/nav/src/store/modules/roverState.ts

```diff
@@ -38,11 +52,32 @@ const state:RoverState = {
     total_wps: 0
   },
 
+  noiseSetttings: {
+    locNoise: {
+      headingStdev: 5,
+      latLonStdev: 1
+    },
```

Comment:
@rzf16 Can you help me choose reasonable values for these?

Reply:
Sorry I'm getting to this late, I wasn't checking my email. What are the units you're using? For bearing, I think a standard deviation of ~5 degrees is safe. For latitude longitude, standard deviation of ~2 meters is reasonable.